src/CommunityToolkit.Aspire.Hosting.ClamAv/ClamAvBuilderExtensions.cs

```diff
@@ -0,0 +1,78 @@
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Utils;
+
+namespace Aspire.Hosting;
+
+/// <summary>
+/// Provides extension methods for adding ClamAv resources to the application model.
+/// </summary>
+public static class ClamAvBuilderExtensions
+{
+
+    /// <summary>
+    /// Adds a ClamAv container resource to the application model.
+    /// </summary>
+    /// <param name="builder">The <see cref="IDistributedApplicationBuilder"/>.</param>
+    /// <param name="name">The name of the resource. This name will be used as the connection string name when referenced in a dependency.</param>
+    /// <param name="port">The host port to bind the underlying container to.</param>
+    /// <returns>A reference to the <see cref="IResourceBuilder{T}"/>.</returns>
+    public static IResourceBuilder<ClamAvResource> AddClamAv(
+        this IDistributedApplicationBuilder builder,
+        string name,
```

Comment:
```suggestion
        [ResourceName] string name,
```

Ensures that the analyzer is run